CHANGELOG.md

```diff
@@ -5,8 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+
 ## [Unreleased]
 
+### Changed
+
+- Prepare project structure for `v1.0.0` by having all go code in `pkg/k8sclient/`.
```

Comment:
So by this I meant that `k8sclienttest` should to `pkg/k8sclient/k8sclienttest` and not to `pkg/k8sclienttest` so we don't have anything but `k8sclient` under `/pkg`. But I'm not sure if this is a good idea anymore. But simply making sure that there are no `.go` files under `/pkg` and if there is only a single package then it's named after repo name also makes sense to me now. Maybe even more sense.

Reply:
I did not get this by your suggestion earlier. Let's try to be more clear with these things in the future. :heart 

I think we are good with this regardless.